data/json/items/gun/9mm.json

```diff
@@ -88,6 +93,140 @@
+    "name": { "str": "Glock 43" },
+    "description": "A 9x19mm Glock of diminutive proportions.  Unfortunately it is incompatible with other 9mm Glock magazines.",
+    "weight": "460 g",
+    "volume": "279 ml",
+    "longest_side": "159 mm",
+    "barrel_length": "87 mm",
+    "price": "570 USD",
+    "material": [ "plastic", "steel" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -1 },
+    "dispersion": 480,
+    "pocket_data": [
+      {
+        "magazine_well": "250 ml",
+        "pocket_type": "MAGAZINE_WELL",
+        "item_restriction": [
+          "glock43_6"
+        ]
+      }
+    ]
+  },  
+  {
+    "id": "glock_43x",
+    "//": "This one uses 10 round doublestack mags, which are incompatible with the standard variant",
+    "copy-from": "gun_base_9mm_semi_pistol",
+    "looks_like": "glock_19",
+    "type": "GUN",
+    "name": { "str": "Glock 43X" },
+    "description": "A 9mm compact Glock.  Despite its name, it does not accept the magazines of the Glock 43.",
+    "weight": "461 g",
+    "volume": "279 ml",
+    "longest_side": "165 mm",
+    "barrel_length": "87 mm",
+    "price": "570 USD",
+    "material": [ "plastic", "steel" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -1 },
+    "valid_mod_locations": [ [ "barrel", 1 ], [ "mechanism", 2 ], [ "sights mount", 1 ], [ "stock mount", 1 ] ],
+    "dispersion": 480,
+    "pocket_data": [
+      {
+        "magazine_well": "250 ml",
+        "pocket_type": "MAGAZINE_WELL",
+        "item_restriction": [
+          "glock43/48x_10"
+        ]
+      }
+    ]
+  },
+  {
+    "id": "glock_26",
+    "copy-from": "gun_base_9mm_semi_pistol",
+    "looks_like": "glock_19",
+    "type": "GUN",
+    "name": { "str": "Glock 26" },
+    "description": "A Glock subcompact pistol chambered in 9mm Luger.  Despite its tiny size, it can fit the magazines of its larger brothers.",
+    "weight": "550 g",
+    "volume": "348 ml",
+    "longest_side": "165 mm",
+    "barrel_length": "87 mm",
+    "valid_mod_locations": [ [ "barrel", 1 ], [ "mechanism", 2 ], [ "sights mount", 1 ], [ "stock mount", 1 ] ],
+    "price": "690 USD",
+    "material": [ "plastic", "steel" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -1 },
+    "dispersion": 480,
+    "pocket_data": [
+      {
+        "magazine_well": "250 ml",
+        "pocket_type": "MAGAZINE_WELL",
+        "item_restriction": [
+          "glockmag_10",
+          "glock17_10",
+          "glockmag",
+          "glockbigmag",
+          "glock17_17",
+          "glock17_22",
+          "glock_drum_50rd",
+          "glock_drum_100rd"
+        ]
+      }
+    ]
+  },
+  {
+    "id": "glock_34",
+    "copy-from": "gun_base_9mm_semi_pistol",
+    "type": "GUN",
+    "name": { "str": "Glock 34" },
+    "description": "A large, stylish Glock chambered in 9mm.  Developed for accuracy, this pistol is used by competition shooters, law enforcement, and even the Baba Yaga.",
+    "weight": "630 g",
```

Comment:
Weight, barrel length is wrong. Measurements might be wrong too, but I don't want to check.